src/map/packets.h

```diff
@@ -1425,6 +1425,7 @@ packet(0x96e,clif->ackmergeitems);
 	packet(0x0887,clif->pStoragePassword,0);  // CZ_ACK_STORE_PASSWORD
 	packet(0x0363,clif->pDull); // CZ_JOIN_BATTLE_FIELD
 	packet(0x0436,clif->pDull); // CZ_GANGSI_RANK
+	packet(0x09C1,clif->crimson_marker,10,2,6,8); // ZC_C_MARKERINFO
```

Comment:
need set correct versions for packet 0x09C1. it exists starting from clients:
2013-07-31cRagexe
2013-07-17cRagexeRE
and any zero

also for packet better use struct in clif_crimson_marker